AndroidNativeLoginTemplate/app/src/main/java/com/salesforce/androidnativelogintemplate/MainApplication.kt

```diff
@@ -28,17 +28,20 @@ package com.salesforce.androidnativelogintemplate
 
 import android.app.Application
 import com.google.android.material.color.DynamicColors
+import com.google.android.recaptcha.Recaptcha
+import com.google.android.recaptcha.RecaptchaAction.Companion.LOGIN
+import com.google.android.recaptcha.RecaptchaClient
 import com.salesforce.androidsdk.mobilesync.app.MobileSyncSDKManager
+import com.salesforce.androidsdk.util.SalesforceSDKLogger
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers.Default
+import kotlinx.coroutines.launch
 
 /**
  * Application class for our application.
  */
 class MainApplication : Application() {
 
-    companion object {
-        private const val FEATURE_APP_USES_KOTLIN = "KT"
```

Comment:
This simply moved to the bottom of the file.  As the companion got larger, it made more sense for the logic of `MainApplication` to come first and supporting code after (I thought).